hypha/apply/funds/templates/funds/application_base_landing.html

```diff
@@ -16,6 +16,8 @@ <h3>Thank you for your submission to the {{ ORG_LONG_NAME }}.</h3>
         <p>If you do not receive an e-mail within 15 minutes please check your
 spam folder and contact {{ ORG_EMAIL|urlize }} for further assistance.</p>
         {% with email_context=page.specific %}<p>{{ email_context.confirmation_text_extra|urlize }}</p>{% endwith %}
+
+        {{ extra_text | safe }}
```

Comment:
It should be [richtext](https://docs.wagtail.io/en/stable/topics/writing_templates.html#rich-text-filter) filter instead of `safe`.